pkg/clients/integration/pipelineruns.go

```diff
@@ -189,8 +189,11 @@ func (i *IntegrationController) WaitForBuildPipelineRunToBeSigned(testNamespace,
 			GinkgoWriter.Printf("pipelinerun for Component %s/%s can't be gotten successfully. Error: %v", testNamespace, componentName, err)
 			return false, nil
 		}
-		if pipelineRun.Annotations["chains.tekton.dev/signed"] != "true" {
-			GinkgoWriter.Printf("pipelinerun %s/%s hasn't been signed yet", pipelineRun.GetNamespace(), pipelineRun.GetName())
+		_, signedAnnotationExists := pipelineRun.Annotations["chains.tekton.dev/signed"]
+		GinkgoWriter.Println("signedAnnotationExists %v", signedAnnotationExists)
+		GinkgoWriter.Println("PR Annotations:  ", pipelineRun.Annotations)
+		if !signedAnnotationExists {
+			GinkgoWriter.Printf("pipelinerun %s/%s doesn't contain the chain signed annotation yet", pipelineRun.GetNamespace(), pipelineRun.GetName())
```

Comment:
```suggestion
		if _, signedAnnotationExists := pipelineRun.Annotations["chains.tekton.dev/signed"]; !signedAnnotationExists {
        		GinkgoWriter.Println("PR Annotations:  ", pipelineRun.Annotations)
			GinkgoWriter.Printf("pipelinerun %s/%s doesn't contain the chain signed annotation yet", pipelineRun.GetNamespace(), pipelineRun.GetName())
```

Reply:
we decided to use function `WaitForBuildPipelineRunToBeSigned` instead, 
this function will be removed